scripts/generate

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 const fs = require("fs");
 const path = require("path");
-const config = require("../ligma.config");
+const config = JSON.parse(fs.readFileSync("./ligma.config.json", "utf-8"));
```

Comment:
path!

there is the `/` in there, i think there is another issue right now via windows

Reply:
IIRC, fs stuff handle it correctly
![image](https://user-images.githubusercontent.com/1373867/190866577-3267c7b9-63ec-4707-abc0-eed6250d2cc4.png)